src/components/sharedPageComponents/contents.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Card from './card';
+import search from '../../assets/Images/Search.png';
+
+export default function Contents({ links }) {
+  return (
+    <article>
+      <form>
+        <input
+          className="card-search"
+          placeholder="링크를 검색해 보세요"
+          type="search"
+        ></input>
+        <button type="submit" className="card-search-button">
+          <img
+            src={search}
+            alt="search"
+            className="card-search-icon"
+            width="16"
+            height="16"
+          ></img>
+        </button>
+      </form>
+      <div className="content-wrapper">
+        {links && links.map((el) => <Card link={el} key={el.id}></Card>)}
```

Comment:
```suggestion
        {links && links.map((link) => <Card link={link} key={link.id}></Card>)}
```

links 배열에서 갑자기 el이 나오는건 어색하게 느껴집니다. 코드 예측성을 위해서라도 link를 사용하는것을 추천드립니다.

혹시 el이라고 사용한 이유를 알 수 있을까요?

+ 만약에 el을 사용하시더라도 el 단축어보다는 element 풀네임으로 표현하는게 코드 가독성 측면에서 훨씬 좋습니다.

Reply:
map을 처음 배울 때 element보다 el로 쓰는 걸로 배우다 보니 습관적으로 el로 쓰는 경향이 있는 것 같습니다.
앞으로는 의미를 생각해서 선언해야겠네요!